runtime/doc/lua.txt

```diff
@@ -4734,6 +4734,23 @@ vim.text.indent({size}, {text}, {opts})                    *vim.text.indent()*
 ==============================================================================
 Lua module: vim.ui                                                    *vim.ui*
 
+*vim.ui.get_progress_status.Opts*
```

Comment:
Why does this get generated separately?  Shouldn't this be part of the function doc? like other functions